internal/cli/cli.go

```
@@ -34,11 +36,75 @@ func Run(ctx context.Context, args []string, output, errorsOutput io.Writer) int
+		return 2
+	}
+	if flags.NArg() != 0 {
+		writeError(errorsOutput, errors.New("bootstrap-labels does not accept positional arguments"))
+		return 2
+	}
+	result, err := factory.New(*configPath).BootstrapLabels(ctx)
+	if err != nil {
+		writeError(errorsOutput, err)
+		return 1
+	}
+	fmt.Fprintf(output, "bootstrapped factory labels for %s: %s\n", result.Repository, strings.Join(result.Labels, ", "))
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintf` is not checked

(errcheck)